### PR TITLE
Update aws-sys-dist.mdx

### PR DESCRIPTION
**Pull request by aj-NR**
Please follow [conventional commit standards](https://www.conventionalcommits.org/en/v1.0.0/)
in your commit messages and pull request title.

## Give us some context

* What problems does this PR solve?
* Add any context that will help us review your changes such as testing notes,
  links to related docs, screenshots, etc.
* If your issue relates to an existing GitHub issue, please link to it.

### CODE REVIEW

File: src/content/docs/infrastructure/amazon-integrations/aws-integrations-list/aws-sys-dist.mdx [hunk 1]
```diff
@@ -38,6 +38,13 @@ New Relic also provides automated process for monitoring of EC2 and can be found
...
 }
 
+```
+Sample variables can also be specified in **Additional Arguments** as shown below. If not specified, they will take the [default values](https://github.com/newrelic/infrastructure-agent/blob/master/assets/examples/infrastructure/newrelic-infra-template.yml.example).
+
+```
+
+{ "SSM_NR_LICENSE_KEY" : "YOUR_NEWRELIC_LICENSE_KEY", "SSM_NRIA_METRICS_NETWORK_SAMPLE_RATE" : "SOME_INTEGER_VALUE", "SSM_NRIA_METRICS_PROCESS_SAMPLE_RATE" : "SOME_INTEGER_VALUE", "SSM_NRIA_METRICS_STORAGE_SAMPLE_RATE" : "SOME_INTEGER_VALUE", "SSM_NRIA_METRICS_SYSTEM_SAMPLE_RATE" : "SOME_INTEGER_VALUE", "SSM_NRIA_METRICS_NFS_SAMPLE_RATE" : "SOME_INTEGER_VALUE" }
+
 ```
```
Comment (ally-sassman) [inline src/content/docs/infrastructure/amazon-integrations/aws-integrations-list/aws-sys-dist.mdx:48]:
```suggestion
```
{ "SSM_NR_LICENSE_KEY" : "YOUR_NEWRELIC_LICENSE_KEY", "SSM_NRIA_METRICS_NETWORK_SAMPLE_RATE" : "SOME_INTEGER_VALUE", "SSM_NRIA_METRICS_PROCESS_SAMPLE_RATE" : "SOME_INTEGER_VALUE", "SSM_NRIA_METRICS_STORAGE_SAMPLE_RATE" : "SOME_INTEGER_VALUE", "SSM_NRIA_METRICS_SYSTEM_SAMPLE_RATE" : "SOME_INTEGER_VALUE", "SSM_NRIA_METRICS_NFS_SAMPLE_RATE" : "SOME_INTEGER_VALUE" }
```
```

**Review by ghost — DISMISSED** [review]

**Comment by ally-sassman** [comment]
@aj-NR Can you please sign the CLA?

File: src/content/docs/infrastructure/amazon-integrations/aws-integrations-list/aws-sys-dist.mdx (hunk 1 above)
Comment (aj-NR) [reply src/content/docs/infrastructure/amazon-integrations/aws-integrations-list/aws-sys-dist.mdx]: addressed

**Comment by aj-NR** [comment]
> @aj-NR Can you please sign the CLA?

Signed it @ally-sassman

**Review by ghost — APPROVED** [review]

**Pull request merged**